demo/allegro5/main.c

```diff
@@ -145,7 +145,7 @@ int main(void)
         nk_input_end(ctx);
 
         /* GUI */
-        if (nk_begin(ctx, "Demo", nk_rect(50, 50, 200, 200),
+        if (nk_begin(ctx, nk_slicez("Demo"), nk_rect(50, 50, 200, 200),
```

Comment:
Shouldn't the demos written in **C** then use the non-slice API (to not scare people off)?

Reply:
Yes yes. I only update the header so we can discuss the interface before i go on a thousand line journey.